lib/apiservers/portlayer/swagger.json

```diff
@@ -3875,4 +3881,4 @@
 			}
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
I gave up on this... Not sure why it keeps getting added in but shouldn't hurt anyone and I wanted to look at the lost+found issue